### DISCUSSION TITLE
Berry Farming minigame side thing

### DISCUSSION THREAD

**Discussion by TheMysterys** [jaylawl/Jaygrounds#6] [Ideas]
While this likely won't get added it came to mine after seeing the Stocks system you have been working on and from the stuff you mentioned in DMs

I was thinking of there being a mini golf course like Feather Farm but based around berries and being able to farm berries for Golf Coins. Depending on how many berries people are selling the price could fluctuate and there be other ties, like random times where crops would die due to a wild weather event or other random things which changes the amount of berry bushes that can be harvested.

Anyway hope this is an interesting idea (although definitely on the crazy side)

**Comment by jaylawl** [comment]
I like it! Do you think it would be possible to incorporate other resources rather than just the berries? Smells like potential

**Reply by TheMysterys** [reply]
Could have it be the same on feather farm where you can harvest the wheat from around the farm, or say mining crystals in Crackle Caves and stuff

Just have different side minigames at some of the courses to just have a extra way to earn Golf Coins
